src/main/java/me/jellysquid/mods/sodium/client/world/biome/BiomeColorCache.java

```diff
@@ -49,26 +49,47 @@ public void update(ChunkRenderContext context) {
         this.maxY = (context.getOrigin().getMaxY() + NEIGHBOR_BLOCK_RADIUS);
         this.maxZ = (context.getOrigin().getMaxZ() + NEIGHBOR_BLOCK_RADIUS) + this.blendRadius;
 
-        Arrays.fill(this.populatedSlices, false);
+        for (boolean[] p : this.populatedSlices.values()) {
+            Arrays.fill(p, false);
+        }
```

Comment:
Optimizing performance by using an incremental flag (without loops) to check for populated status instead of relying on two nested for-loops.